src/plugins/email-smtp/util/getMailConfig.js

```diff
@@ -119,21 +53,7 @@ export default async function getMailConfig(context, shopId) {
   if (MAIL_URL) return getConfigFromMailUrl({ logger, mailUrl: MAIL_URL });
```

Comment:
As far as I can tell, `getConfigFromMailUrl` does not do anything with `logger` so I see no reason to pass it in. It can be added to the returned config object before returning it from this function.

Also, since ENV should not change while the server is running, `getConfigFromMailUrl` could be called just one time in top-level code, and then the config object simply returned by this function.

Reply:
refactored